pytensor/tensor/math.py

```diff
@@ -277,6 +278,31 @@ def grad(self, inp, grads):
         return [x.zeros_like()]
 
 
+def argmax(x, axis=None, keepdims=False):
+    """
+    Returns indices of maximum elements obtained by iterating over given axis.
+
+    When axis is None (the default value), the argmax is performed
+    over the flattened tensor.
+
+    Parameters
```

Comment:
x and axis are also parameters and should have (perfunctory) docstrings

Reply:
:( when refactoring existing function and they ask for more docstrings. My fault I guess :D